rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -2146,22 +2222,35 @@ extern void ppu_precompile(std::vector<std::string>& dir_queue)
 
 	g_progr = "Compiling PPU modules";
```

Comment:
This should probably be renamed, as it is exactly the same text as the one for the main modules.

Reply:
I think previous message (about scanning) was added to make dialog appear earlier, but it doesn't matter so much.